GOVERNANCE.md

```diff
@@ -225,6 +227,37 @@ Additionally, there is a strict process for assigning Project-Funded Bounties:
   - If an objection is raised, the request is put on hold until a Lead makes the final decision.
 - There is a maximum amount of issues with pledges someone can have assigned to them. The limit is 2 for core contributors, and 1 for anyone else. By limiting the amount of issues with pledges that someone can have assigned, we make sure the bounties remain available for others to pick up.
 
+### Paid Contracting
+
+Core Contributors may enter freelance contracts with clients to work on Biome. Such contracts are
+between the contributor and their client, so they fall mostly outside the responsibility of the
+Biome project. Nevertheless, we can explicitly endorse such contracts under the following
+conditions:
+
+- Core Contributors that are open to contracts may be advertised on the project website if they
+  please.
+- Clients that hire a Core Contributor for an extended period (3 months or more) are eligible to
+  the same benefits as a project sponsor. Their sponsorship benefits will be based on the monthly
+  fee paid to the Core Contributor.
+- If the work that is expected to be delivered does not contribute to Biome's last-published
+  roadmap, Biome asks for a 30% fee over the total amount earned through the contract. This is to
```

Comment:
```suggestion
  roadmap, Biome asks for a 30% fee over the total amount earned through the contract (Total = Contractor Rate - VAT). This is to
```
Just in case it isn't clear enough. What do you think? 

I'm not sure if we can take VAT money, because contractors may want to entire VAT for their own bookkeeping, so we should "charge" the "net" earnings.

Reply:
Good point :+1: